inference-engine/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -25,15 +25,15 @@ set(DEPENDENCIES
     sharedTestClasses
 )
 
-if (NGRAPH_ONNX_IMPORT_ENABLE)
+if (NGRAPH_ONNX_IMPORT_ENABLE AND NOT NGRAPH_USE_PROTOBUF_LITE)
     list(APPEND INCLUDES "${OpenVINO_MAIN_SOURCE_DIR}/docs/onnx_custom_op")
     list(APPEND LINK_LIBRARIES onnx_custom_op)
     list(APPEND DEPENDENCIES onnx_custom_op)
 else()
     set(EXCLUDED_SOURCE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/onnx_reader")
 endif()
 
-if (NOT NGRAPH_ONNX_IMPORT_ENABLE OR NOT ENABLE_MKL_DNN)
+if (NOT NGRAPH_ONNX_IMPORT_ENABLE OR NOT ENABLE_MKL_DNN OR (NGRAPH_ONNX_IMPORT_ENABLE AND NGRAPH_USE_PROTOBUF_LITE))
```

Comment:
can we just use `OR NGRAPH_USE_PROTOBUF_LITE` ?

Reply:
that's right, done